main.tf

```
@@ -1,80 +1,80 @@
 //
 // Module: inspector
 //
-provider "aws" {}
+provider "aws" {
+}
 
 locals {
-  iam_role_specified = "${var.iam_role_arn == ""}"
-  iam_role_arn       = "${local.iam_role_specified ? join("", aws_iam_role.this.*.arn) : var.iam_role_arn}"
+  iam_role_specified = var.iam_role_arn == ""
```

per the changes requested [here](https://github.com/plus3it/terraform-aws-vault/pull/1#discussion_r319090385) recommending:
```suggestion
  iam_role_specified = var.iam_role_arn == null
```